sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -451,6 +451,12 @@ def analyze_sentiment(  # type: ignore
         :type documents:
             list[str] or list[~azure.ai.textanalytics.TextDocumentInput] or
             list[dict[str, str]]
+        :keyword bool show_opinion_mining: Whether to mine the opinions of a sentence and conduct more
+            granular analysis around the aspects of a product or service (also known as
+            aspect-based sentiment analysis). If set to true, the returned
+            :class:`~azure.ai.textanalytics.SentenceSentiment` objects
+            will have property `mined_opinions` containing the result of this analysis. Only available for
```

Comment:
Not sure if we should say this here, as we will have to update it once service GA 3.1
We could just use the link to service docs and let them manage the versions of where the functionality is available

Reply:
Agreed that it is an extra step of maintenance, but I think we should still have this documentation is here, so it's easier for users to see off the bat that this is only available in certain versions.

Will follow your guidance and open an issue to track this for the future, thanks!

Reply:
issue here: https://github.com/Azure/azure-sdk-for-python/issues/12832